src/views/admin/PersonDetailsView.vue

```diff
@@ -2051,10 +2050,36 @@
                       ></v-icon>
                     </v-col>
                     <v-col>
-                      <template v-if="twoFactorAuthenticationConnectionError">
-                        <p>
-                          {{ twoFactorAuthenticationConnectionError }}
-                        </p>
+                      <template
+                        v-if="twoFactorAuthentificationStore.errorCode && !twoFactorAuthentificationStore.loading"
+                      >
+                        <v-row v-if="twoFactorAuthentificationStore.errorCode === 'PI_UNAVAILABLE_ERROR'">
+                          <p
+                            class="text-body"
+                            data-testid="connection-error-text"
+                          >
+                            {{ $t('admin.person.twoFactorAuthentication.errors.connection') }}
+                          </p>
+                        </v-row>
+                        <v-row v-else-if="twoFactorAuthentificationStore.errorCode === 'TOKEN_STATE_ERROR'">
+                          <p
+                            class="text-body"
+                            data-testid="token-state-error-text"
+                          >
+                            <i18n-t
+                              keypath="admin.person.twoFactorAuthentication.errors.tokenStateError"
+                              for="admin.person.twoFactorAuthentication.errors.iqshHelpdesk"
+                              tag="label"
+                            >
+                              <a
+                                :href="$t('admin.person.twoFactorAuthentication.errors.iqshHelpdeskLink')"
+                                rel="noopener noreferrer"
+                                target="_blank"
+                                >{{ $t('admin.person.twoFactorAuthentication.errors.iqshHelpdesk') }}</a
+                              >
+                            </i18n-t>
+                          </p>
+                        </v-row>
```

Comment:
Was ist der Purpose vom Switch case, wenn wir auch in der Template eine Fallunterscheidung machen?

```
    switch (twoFactorAuthentificationStore.errorCode) {
      case 'TOKEN_STATE_ERROR':
        return t('admin.person.twoFactorAuthentication.errors.tokenStateError');
      case 'PI_UNAVAILABLE_ERROR':
        return t('admin.person.twoFactorAuthentication.errors.connection');
      default:
        return '';
```

Reply:
eigentlich brauch ich den nur um Sachen ein und auszublenden- Theoretisch brauche ich auch nicht den Text direkt sondern nur ob es einen error gibt mit den beiden error codes